package.json

```
@@ -8,6 +8,7 @@
   "dependencies": {
     "accepts": "~1.3.7",
     "batch": "0.6.1",
+    "bytes": "^3.1.2",
```

```suggestion
    "bytes": "~3.1.2",
```

The `^` was not supported in old versions of Node like 0.8, this will fix it. In the next major we can include the `^` as we will drop support to legacy versions.